castle/cms/tiles/metadata.py

```diff
@@ -85,7 +85,7 @@ def get_basic_tags(self):
                 'modificationDate': _date(context, 'modified'),
                 'publicationDate': _date(context, 'effective'),
                 'expirationDate': _date(context, 'expires'),
-                'generator': 'CastleCMS 2.0',
+                'generator': 'CastleCMS 2.5.17',
```

Comment:
is this a way for this to be dynamic so we don't have to remember to update it each time?